extension-manifest-v3/src/background/utils/trackerdb.js

```diff
@@ -54,9 +45,8 @@ export async function getMetadata(request) {
 
 const patterns = new Map();
 export async function getPattern(key) {
-  if (trackerDBStartupPromise) {
-    await trackerDBStartupPromise;
```

Comment:
this is a breaking change - why don't we wait for trackerdb to be ready?

Reply:
I added await for the init promise.